tests/test_hammerdb.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+source tests/common.sh
+
+function initdb_pod {
+	echo "Setting up a MS-SQL DB Pod"
+	kubectl create -f tests/mssql.yaml
+}
+
+function finish {
+	echo "Cleaning up hammerdb"
+	kubectl delete -f tests/mssql.yaml 
+	kubectl delete -f tests/test_crs/valid_hammerdb.yaml
+	delete_operator
+}
+
+trap finish EXIT
+
+function functional_test_hammerdb {
+	initdb_pod
+	sleep 30
+	apply_operator
+	kubectl apply -f tests/test_crs/valid_hammerdb.yaml
+	hammerdb_pod=$(get_pod 'app=hammerdb' 300)
```

Comment:
This needs to be `uuid=$(get_uuid 20)` and then the next couple of lines should make use of $uuid

Reply:
ok, that should be in now